sdk/src/Handlers/EntityFramework/EFInterceptor.cs

```diff
@@ -80,7 +80,11 @@ public override DbDataReader ReaderExecuted(DbCommand command, CommandExecutedEv
         /// <param name="result">Result from <see cref="IInterceptor"/>.</param>
         /// <param name="cancellationToken">Instance of <see cref="CancellationToken"/>.</param>
         /// <returns>Task representing the async operation.</returns>
+#if NET5_0
+        public override ValueTask<InterceptionResult<DbDataReader>> ReaderExecutingAsync(DbCommand command, CommandEventData eventData, InterceptionResult<DbDataReader> result, CancellationToken cancellationToken = default)
```

Comment:
Is it better to create a new interceptor class for .NET 5.0, similar as `EFInterceptor.net5.cs` ?